test/functional/wallet_maxtxfee.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test wallet maxtxfee limiting in conjunction with settxfee or mintxfee."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_raises_rpc_error
+
+class WalletMaxFeeTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+        self.setup_clean_chain = True
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def run_test(self):
+        self.nodes[0].generate(105)
+
+        outputs = {}
+        
```

Comment:
```suggestion

```

Reply:
Fixed the linting errors, sorry about that. I managed to figure out how to run these locally before I commit now. So this shouldn't happen to me again.